ci-operator/step-registry/openshift-qe/ipsec-metrics-validation/openshift-qe-ipsec-metrics-validation-commands.sh

```
@@ -0,0 +1,258 @@
+echo "Testing pod-to-pod connectivity across all nodes..."
+
+# Get all nginx pod IPs and nodes
+oc get pods -n ${TEST_NS} -l app=nginx-test -o json | \
+  jq -r '.items[] | "\(.status.podIP) \(.spec.nodeName) \(.metadata.name)"' > /tmp/nginx_pods.txt
+
+TOTAL_TESTS=0
+PASSED_TESTS=0
+FAILED_TESTS=0
+
+while IFS= read -r source_line; do
+    SOURCE_IP=$(echo ${source_line} | awk '{print $1}')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)